### PR TITLE
Refactor some abductor code using the now removed anim function

### PR DESCRIPTION
**Pull request by optimumtact**
These were not caught as they used addtimer and text proc names
(booo!)

### CODE REVIEW

**Comment by optimumtact** [comment]
Also removed some src that were not useful

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 1]
```diff
@@ -50,11 +50,9 @@
...
 		return
 	stealth_active = 1
-	if(istype(src.loc, /mob/living/carbon/human))
-		var/mob/living/carbon/human/M = src.loc
-		addtimer(GLOBAL_PROC, "anim", 0, FALSE, M.loc, M, 'icons/mob/mob.dmi',
-			null, "cloak", null, M.dir)
-
+	if(istype(loc, /mob/living/carbon/human))
+		var/mob/living/carbon/human/M = loc
+		new /obj/effect/overlay/temp/dir_setting/abductor_cloak(get_turf(M), M.dir)
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: Just use `/obj/effect/overlay/temp/dir_setting/ninja/cloak` instead

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 2]
```diff
@@ -65,10 +63,9 @@
 	if(!stealth_active)
 		return
 	stealth_active = 0
-	if(istype(src.loc, /mob/living/carbon/human))
-		var/mob/living/carbon/human/M = src.loc
-		addtimer(GLOBAL_PROC, "anim", 0, FALSE, M.loc, M, 'icons/mob/mob.dmi',
-			null, "uncloak", null, M.dir)
+	if(istype(loc, /mob/living/carbon/human))
+		var/mob/living/carbon/human/M = loc
+		new /obj/effect/overlay/temp/dir_setting/abductor_uncloak(getturf(M), M.dir)
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: Just use `/obj/effect/overlay/temp/dir_setting/ninja` instead

File: code/game/objects/effects/overlays.dm [hunk 3]
```diff
@@ -157,6 +157,14 @@
 /obj/effect/overlay/temp/dir_setting/wraith/out
 	icon_state = "phase_shift"
 
+/obj/effect/overlay/temp/dir_setting/abductor_cloak
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "cloak"
+
+/obj/effect/overlay/temp/dir_setting/abductor_uncloak
```
Comment (ChangelingRain) [inline code/game/objects/effects/overlays.dm]: Accordingly, you should remove both of these

**Review by ChangelingRain — COMMENTED** [review]
copypaste is the devil

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: oh, and this should be `PoolOrNew(/obj/effect/overlay/temp/dir_setting/ninja/cloak, list(get_turf(M), M.dir))`

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: `PoolOrNew(/obj/effect/overlay/temp/dir_setting/ninja, list(get_turf(M), M.dir))`

**Comment by optimumtact** [comment]
Why do your review comments still remain when the code has been updated? Annoying

**Comment by Ninja137** [comment]
Have you tried removing them?

**Pull request merged**